packages/material-ui/src/Grow/Grow.js

```diff
@@ -27,22 +27,21 @@ const styles = {
  * [Popover](/utils/popover/) components.
  * It uses [react-transition-group](https://github.com/reactjs/react-transition-group) internally.
  */
-class Grow extends React.Component {
-  componentWillUnmount() {
-    clearTimeout(this.timer);
-  }
+function Grow(props) {
```

Comment:
For another pull request, we can forward the ref.

Reply:
We will be able to remove withTheme too :)

Reply:
There is nothing to forward the ref to. It just renders children.

Reply:
Interesting, we spread the props but not the ref to the child element. Does it sounds good?

Reply:
OK, it's fine by me.

Reply:
All the transition components are very suspicious to me at the moment. There's just a lot of "stuff" going on. I do want to refactor them at some point.